### ISSUE TITLE
ability to disable 5x5 restriction on zoom target?

### ISSUE THREAD

**Issue by wchristian** [hansmaad/chartist-plugin-zoom#23]
We're zooming on a time series and absolutely don't care about vertical zoom, so we use the callback to revert the vertical part after each zoom. That means that the 5px vertical restriction on the zoom activating is completely meaningless for us, as we want to only draw horizontal lines anyhow. Any chance of adding an option to disable that or at least converting those two numbers from magical numbers to something that can be modified?

**Comment by hansmaad** [comment]
I think we could add this as an option. Any naming suggestions? I think it's some kind of `zoomThreshold` (if I remember correctly...) Could be a number or `{x: number, y: number}` (as I understood you want to specify different values for vertical and horizontal?)

**Comment by wchristian** [comment]
zoomActivationThreshold would be a little more accurate, but i honestly have no particular preference there, and will be happy with what you choose. :)

And yeah, having it accept both a single number, or an object would be the most DWIM for the matter.

And accurate, we would want to entirely disable the vertical check, and make the horizontal check be only 2 pixels.

**Comment by hansmaad** [comment]
This should then be expressed as 
```
const options = {
  zoomActivationThreshold: { x: 2, y: 0 }
};
```
Correct?

**Comment by hansmaad** [comment]
```
const options = {
  zoomActivationThreshold: { x: 2, y: 0 }
};
```
available in 0.6.0
